xunit.runner.visualstudio.testadapter/VsTestRunner.cs

```diff
@@ -147,8 +147,9 @@ static ITestCase Deserialize(LoggerHelper logger, ITestFrameworkExecutor executo
                 {
                     var reporterMessageHandler = new DefaultRunnerReporter().CreateMessageHandler(new VisualStudioRunnerLogger(logger));
 
-                    foreach (var assemblyFileName in sources)
+                    foreach (var assemblyFileNameCanBeWithoutAbsolutePath in sources)
                     {
+                        var assemblyFileName = Path.GetFullPath(assemblyFileNameCanBeWithoutAbsolutePath);
```

Comment:
Path.GetFullPath is not available on all platforms. You will need to use an `#if` to only call this when running on the desktop CLR.